### PR TITLE
feat: Fir 15408 new output format stbu

### PR DESCRIPTION
**Pull request by stepansergeevitch**
Switched to support new `JSON_Compact` output format. Fixed unit and integration tests
https://github.com/firebolt-db/firebolt-go-sdk/actions/runs/4064398318

### CODE REVIEW

**Comment by ptiurin** [comment]
What's `stbu` in the title?

**Review by ptiurin — APPROVED** [review]
LGTM, thanks for cleaning up the tests, much more readable now

**Comment by stepansergeevitch** [comment]
> What's `stbu` in the title?

Short for Stepan Burlakov. I believe I've accidentally added this, it was a rule in my previous company :)

File: rows_test.go [hunk 1]
```diff
@@ -18,52 +18,59 @@ func assert(test_val interface{}, expected_val interface{}, t *testing.T, err st
...
+        	{"name":"text_col","type":"text"},
+        	{"name":"date_col","type":"date"},
+        	{"name":"timestamp_col","type":"timestamp"},
+        	{"name":"pgdate_col","type":"pgdate"},
+        	{"name":"timestampntz_col","type":"timestampntz"},
+        	{"name":"timestamptz_col","type":"timestamptz"},
+        	{"name":"legacy_bool_col","type":"int"},
+        	{"name":"array_col","type":"array(int)"},
+        	{"name":"nested_array_col","type":"array(array(text))"},
+        	{"name":"new_bool_col","type":"boolean"}],
```
Comment (aymeric-dispa) [inline rows_test.go:38]: I think it might be worth testing Decimal with scale/precision and array of decimal (like: array(Decimal(38, 30)) )

**Review by aymeric-dispa — APPROVED** [review]

**Pull request merged**